packages/terra-open-source-scripts/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removed
```

Comment:
Technically non passive. Though for our uses, I'm not sure if it will affect us since for us most, if not all, packages that use the open source scripts, also use terra-functional testing.

Reply:
I searched in github for all the places that depend on @cerner/terra-open-source-scripts. There are a few but they all already also depend on @cerner/terra-functional-testing so this should continue to work for them.